pkgs/tools/misc/pv/default.nix

```diff
@@ -6,11 +6,11 @@
 
 stdenv.mkDerivation rec {
   pname = "pv";
-  version = "1.6.20";
+  version = "1.7.24";
 
   src = fetchurl {
-    url = "https://www.ivarch.com/programs/sources/pv-${version}.tar.bz2";
-    sha256 = "00y6zla8h653sn4axgqz7rr0x79vfwl62a7gn6lzn607zwg9acg8";
+    url = "https://www.ivarch.com/programs/sources/pv-${version}.tar.gz";
+    sha256 = "sha256-O/Q8WAnI1QBm6urqWhFfZQPFejjBUZdbcQqivuhXtl4=";
   };
 
   patches = [
```

Comment:
I think this patch is not needed anymore.

Looks like the issue has been fixed with these commits:

* https://codeberg.org/a-j-wood/pv/commit/f13f5e70c1b84cd748cd9e12d55251d78449e1b6
* https://codeberg.org/a-j-wood/pv/commit/7448c7f320d25c43dfd3da8d126a26ec65c33d3e